installLoadTestServer.rst

```diff
@@ -0,0 +1,97 @@
+===========================
+Install Load Test  
+===========================
+
+
+The *loadtest* is installed atop an existing Mattermost installation. If you do not have Mattermost installed along with its ancillary componets then the installations in  must be completed.
+
+
+Prerequisites to Load Test
+===========================================
+
+From the `Mattermost Administrator's Guide <https://docs.mattermost.com/guides/administrator.html>`_ follow the instructions for a standard install. 
+
+Once completed the basic installation follow the instructions for the *loadtest* install defined below.
+
+Load Test Install
+==========================================
+
+The following steps define the *loadtest* install.
+
+
+1. Install the build-essential package.
+
+    ``sudo apt-get install build-essential``
+    
+Linux
+--------------
+
+1. Open a ``Terminal`` and download the Go 1.8 binary file for Linux with the following command:
```

Comment:
We don't want to keep updating this for the latest go. Step 1 can just be install golang: https://golang.org/dl/